distribution/ddf-common/src/main/resources/etc/artemis-backup.xml

```diff
@@ -0,0 +1,171 @@
+         Your system could perform 25 writes per millisecond
+         on the current journal configuration.
+         That translates as a sync write every 40000 nanoseconds
+        -->
+        <journal-buffer-timeout>40000</journal-buffer-timeout>
+
+        <message-expiry-scan-period>1000</message-expiry-scan-period>
+
+        <!-- how often we are looking for how many bytes are being used on the disk in ms -->
+        <disk-scan-period>5000</disk-scan-period>
+
+        <!-- once the disk hits this limit the system will block, or close the connection in certain protocols
+       that won't support flow control. -->
+        <max-disk-usage>90</max-disk-usage>
+
+        <!-- the system will enter into page mode once you hit this limit.
+             This is an estimate in bytes of how much the messages are using in memory -->
+        <global-max-size>504857600</global-max-size>
+
+        <ha-policy>
+            <replication>
+                <slave/>
+            </replication>
+        </ha-policy>
+
+        <!-- @formatter:off-->
+        <!-- Connectors -->
+        <connectors>
+            <connector name="netty-live-connector">tcp://${artemis.live.host}:${artemis.multiprotocol.port}</connector>
+            <connector name="netty-connector">tcp://${artemis.backup.host}:${artemis.multiprotocol.port}</connector>
+            <connector name="vm-connector">vm://0</connector>
+        </connectors>
+
+        <!-- Acceptors -->
+        <acceptors>
+            <!-- Default ActiveMQ Artemis Acceptor.  Multi-protocol adapter.  Currently supports Core, OpenWire, Stomp and AMQP. -->
+            <!-- performance tests have shown that openWire performs best with these buffer sizes -->
+            <acceptor name="artemis">tcp://0.0.0.0:${artemis.multiprotocol.port}?tcpSendBufferSize=1048576;tcpReceiveBufferSize=1048576;sslEnabled=true;keyStorePath=${javax.net.ssl.keyStore};keyStorePassword=${javax.net.ssl.keyStorePassword};enabledCipherSuites=${https.cipherSuites};enabledProtocols=${https.protocols}</acceptor>
+
+            <acceptor name="openwire">tcp://0.0.0.0:${artemis.openwire.port}?tcpSendBufferSize=1048576;tcpReceiveBufferSize=1048576;?sslEnabled=true;keyStorePath=${javax.net.ssl.keyStore};keyStorePassword=${javax.net.ssl.keyStorePassword};enabledCipherSuites=${https.cipherSuites};enabledProtocols=${https.protocols};protocols=OPENWIRE</acceptor>
+            <!-- AMQP Acceptor.  Listens on default AMQP port for AMQP traffic.-->
+            <acceptor name="amqp">tcp://0.0.0.0:${artemis.amqp.port}??sslEnabled=true;keyStorePath=${javax.net.ssl.keyStore};keyStorePassword=${javax.net.ssl.keyStorePassword};enabledCipherSuites=${https.cipherSuites};enabledProtocols=${https.protocols};protocols=AMQP</acceptor>
+
+            <acceptor name="vm">vm://0</acceptor>
+        </acceptors>
+        <!-- @formatter:on  -->
+
+        <cluster-connections>
+            <cluster-connection name="my-cluster">
```

Comment:
should we rename this to something more descriptive? I've run into issues before where the names conflict or don't really make sense